src/model/entities/test/rundown.spec.ts

```diff
@@ -9,8 +9,9 @@ import { NotActivatedException } from '../../exceptions/not-activated-exception'
 import { NotFoundException } from '../../exceptions/not-found-exception'
 import { LastPartInSegmentException } from '../../exceptions/last-part-in-segment-exception'
 import { LastPartInRundownException } from '../../exceptions/last-part-in-rundown-exception'
+import { AlreadyActivatedException } from '../../exceptions/already-activated-exception'
 
-describe('Rundown', () => {
+describe(`${Rundown.name}`, () => {
```

Comment:
As per Anders comment on #10, string interpolation can be removed. 
Same goes for a number of other describes.